firmware/integration/rusefi_config.txt

```diff
@@ -467,7 +467,7 @@ bit useTLE8888_cranking_hack;
 bit kickStartCranking
 bit useSeparateIdleTablesForCrankingTaper;This uses separate ignition timing and VE tables not only for idle conditions, also during the postcranking-to-idle taper transition (See also afterCrankingIACtaperDuration).
 bit launchControlEnabled;
-bit doNotFilterTriggerEdgeNoise,"without filter","with filter";"Detect double trigger edges"
```

Comment:
Can we please keep the bit? Implementing the logic would also be great!

Reply:
Do we think there is any value in turning it off? It's computationally very very cheap, and I don't think has any risk in it always being on. If we can be smarter (ie, detect that error) and it's not dangerous to do, why shouldn't we just always do it?

Reply:
The common pattern I see is "we should be smart by default but have advanced feature to act simpler"